app/controllers/course/video/submission/submissions_controller.rb

```diff
@@ -10,11 +11,18 @@ def index
   end
 
   def create
-    if @submission.save
-      redirect_to edit_course_video_submission_path(current_course, @video, @submission)
+    make_and_edit_submission
+  end
+
+  def new
```

Comment:
Seems like the end goal of this endpoint is to redirect to the edit submission page. If the behaviour if new is non-standard, I would rather rename it to a non-REST-ful route, eg. `attempt` or something more relevant.

Reply:
Alright, sounds good!